src/documents/consumer.py

```diff
@@ -191,6 +200,9 @@ def _ocr(self, pngs, lang):
         Performs a single OCR attempt.
         """
 
+        if not pngs:
```

Comment:
I'm a little worried about this bit.  If `pngs` is empty, then we should be blowing up somewhere, not silently returning `""`, no?  Did you run into a case where this happened?  Perhaps we can fix the cause rather than handle a case where this would happen?